examples/subplots_axes_and_figures/secondary_axis.py

```diff
@@ -0,0 +1,105 @@
+"""
+==============
+Secondary Axis
+==============
+
+Sometimes we want as secondary axis on a plot, for instance to convert
+radians to degrees on the same plot.  We can do this by making a child
+axes with only one axis visible via `.Axes.axes.secondary_xaxis` and
+`.Axes.axes.secondary_yaxis`.
+
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+from matplotlib.transforms import Transform
+from matplotlib.ticker import (
+    AutoLocator, AutoMinorLocator)
+
+fig, ax = plt.subplots(constrained_layout=True)
+x = np.arange(0, 360, 1)
+y = np.sin(2 * x * np.pi / 180)
+ax.plot(x, y)
+ax.set_xlabel('angle [degrees]')
+ax.set_ylabel('signal')
+ax.set_title('Sine wave')
+
+secax = ax.secondary_xaxis('top', conversion=[np.pi / 180])
+secax.set_xlabel('angle [rad]')
+plt.show()
+
+###########################################################################
+# The conversion can be a linear slope and an offset as a 2-tuple.  It can
```

Comment:
Would it make sense to explicitely show the use of this (slope,offset)?  I think this is somehow the most common case, or at least the one where there is already a Celsius->Fahrenheit example in the gallery (...which could then be replaced?).